sdk-tests/src/test/java/io/dapr/it/spring/messaging/DaprSpringMessagingIT.java

```diff
@@ -86,6 +87,7 @@ public void beforeEach() throws InterruptedException {
   }
 
   @Test
+  @Disabled("Test is flaky due to global state in the spring test application.")
```

Comment:
@salaboy This test is flaky, it has global state for setting properties somewhere. I disabled it until we find it. Can you help in a new PR after this?

Reply:
Created a tracking issue.